ci-operator/config/redhat-developer/helm/redhat-developer-helm-release-3.11__helm-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+binary_build_commands: make -f openshift-ci/Makefile build
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tests
+images:
+- dockerfile_path: openshift-ci/Dockerfile.tests
+  from: base
+  to: helm
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: helm-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+    test:
+    - as: helm-acceptance-test
+      commands: make -f openshift-ci/Makefile build test-acceptance
```

Comment:
There are not any XML results files being saved as artifacts for this testing. This is important so that we know how many test were run and when failures occur we can find the error from the XML itself.

[This](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/39808/rehearse-39808-periodic-ci-redhat-developer-helm-release-3.11-helm-ocp4.14-lp-interop-helm-interop-aws/1663798953868529664/artifacts/helm-interop-aws/helm-acceptance-test/) is where the artifacts for this test stanza should appear.

Here is [an example](https://github.com/openshift/release/blob/master/ci-operator/step-registry/odo/tests/odo-tests-commands.sh#L12) of copying test XMLs to the proper location. 

There is an $ARTIFACT_DIR automatically available which will save files in the artifacts linked above. We also need these files to start with `junit_`.

Reply:
We can add these commands in the `test` section of the config job itself right?.